lgc/test/lgc-tool.lgc

```diff
@@ -0,0 +1,39 @@
+; Simple test that checks that the lgc command works with multiple modules in
```

Comment:
This could be put in this directory as a a tutorial sample (sample.lgc). Do you have any plan on the naming schema of those tests (or maybe hierarchy on different GPUs)? This will quickly help the developers to determine if there are tests existing. The purposes are two: (1) Avoid add new tests and just extend existing ones (2) check if a test is missing and should be added as a new one. I don't like the naming in LLVM tests/CodeGen/AMDGPU. Besides the intrinsic names, it is not easy to quickly find a test that serves certain functionalities.

Reply:
This isn't just a sample; it serves a useful purpose for checking that the tool works, especially its -extract option.

I take your point about naming of tests, but I don't really know the answer yet. Let's leave it flat for the moment, and then move tests around later when we have a better idea of what kinds of test we have.